common-files/utils/httputils.mjs

```diff
@@ -109,9 +111,9 @@ const applyAxiosDefaults = () => {
 const getRequestParam = req => {
   const dataInput = {};
 
-  if (Object.keys(req.query).length !== 0) dataInput.query = req.query;
-  if (Object.keys(req.params).length !== 0) dataInput.params = req.params;
-  if (Object.keys(req.body).length !== 0) dataInput.body = req.body;
+  if (req.query && Object.keys(req.query).length !== 0) dataInput.query = req.query;
```

Comment:
```
if (Object.keys(req?.query ?? []).length !== 0)
```

…both do the same. Just a simplification that easily can be ignored.

Reply:
I'll do this in another moment.